frontend/public/components/machine-config-pool.tsx

```diff
@@ -59,8 +67,8 @@ const MachineConfigPoolCharacteristics: React.SFC<MachineConfigPoolCharacteristi
         <dd>
           {
             configuration.source
-              ? _.map(configuration.source, mc =>
-                <ResourceLink kind={machineConfigReference} name={mc.name} title={mc.name} />
+              ? _.map(configuration.source, (mc, index) =>
+                <ResourceLink key={`configuration-source-${index}`} kind={machineConfigReference} name={mc.name} title={mc.name} />
```

Comment:
nit: any reason not to use just use `key={index}`?

Reply:
I thought that was bad practice, but I don't really care that much 😆. I can change it.

Reply:
Yeah, but using the index in a string has the same drawbacks... Unfortunately there's not a good key to use here unless maybe we combine apiVersion+kind+name. I'm not sure it's worth it in this case.

Reply:
Fixed